### DISCUSSION TITLE
Sonoff NsPanel

### DISCUSSION THREAD

**Discussion by mpcpol** [RavenSystem/esp-homekit-devices#1708] [New Features]
any possibility of working on npanel with raven core?

**Comment by Fudalii** [comment]
Ns Panel probably have ESP32 or BL-602L20  (Paring in eWeLink is with Bluetooth)

**Comment by majortom64** [comment]
No that it has been out for a bit, I am also curious about getting it supported under HomeKit. Has anyone one looked at this over the last (almost) two years? :-D
